assemblies/nexus-base-template/src/main/resources/overlay/etc/ehcache.xml

```diff
@@ -1,32 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<ehcache xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://ehcache.org/ehcache.xsd"
-         updateCheck="false"
```

Comment:
Note that this is ehcache3, which doesn't implement the "call home" feature (at least, not yet)